src/mitigation.c

```diff
@@ -78,4 +99,10 @@ void mitigation_deactivate(struct mitigation *m)
 		r = list_entry(node, struct mitigation_resource, list_node);
 		resource_disable(r->resource);
 	}
+
+	// fallback_resources also need to be disabled
+	for_list_node(&m->fallback_resources, node) {
+		r = list_entry(node, struct mitigation_resource, list_node);
+		resource_disable(r->resource);
```

Comment:
Yeah this requires handling, although the same might happen with normal resources, i.e when the enabling fails for some reason. How is it solved there?